### DISCUSSION TITLE
How to use oidc with swagger?

### DISCUSSION THREAD

**Discussion by dkoci** [quarkusio/quarkus#29639] [Community]
Hi.
My api is protected (oidc/Azure AD). Is it possible to login with swagger-ui? OIDC uthentication isntead of Bearer authentication. Is that possible?

Thank you

**Comment by sberyozkin** [comment]
@dkoci 
Hi, it should be possible from Swagger UI itself with its Authorization option and then indeed Swagger UI will interact with Quarkus using the bearer token. If you mean that you'd like Swagger UI to call Quarkus `web-app` and Quarkus itself will drive the authentication with Azure then it is currently not possible.
